sdks/wasm/sdk/Mono.WebAssembly.Build/Mono.WebAssembly.Build.targets

```diff
@@ -81,4 +88,10 @@
 			OutputFile="$(_MonoWasmRuntimeJs)"
 		/>
   </Target>
+	<Target Name = "CleanManaged">
```

Comment:
I would probably give this target a more explicit name and prefix it with an underscore to indicate that users should not call it directly, e.g. `_CleanWasmManagedOutput`

Reply:
Addressed in recent commit